### PR TITLE
Ftr/add configuration setting

### PR DESCRIPTION
**Pull request by nobnisai**

### CODE REVIEW

**Review by dkastl — APPROVED** [review]

**Review by sanak — APPROVED** [review]
LGTM!

File: config/locales/ja.yml [hunk 1]
```diff
@@ -1,3 +1,10 @@
 ja:
   permission_view_gtt_smash: "GTT SMASHの閲覧"
   project_module_gtt_smash: "GTT SMASH"
+
+  tracker_simple_note: "Tracker for Simple Notes"
+  tracker_photo_note: "Tracker for Photo Notes"
+  tracker_gps_log: "Tracker for GPS Logs"
+  default_subject: "Default subject"
+
+  setting_message: "Caution: 	ensure hat selected trackers do not have required custom fields!"
```
Comment (sanak) [inline config/locales/ja.yml]: @nobnisai `ensure hat` may be typo. Probably, `ensure that` ?

File: config/locales/en.yml [hunk 2]
```diff
@@ -1,3 +1,10 @@
 en:
   permission_view_gtt_smash: "View GTT SMASH"
   project_module_gtt_smash: "GTT SMASH"
+
+  field_tracker_simple_note: "Tracker for Simple Notes"
+  field_tracker_photo_note: "Tracker for Photo Notes"
+  field_tracker_gps_log: "Tracker for GPS Logs"
+  field_default_subject: "Default subject"
+  
+  setting_message: "Caution: 	ensure hat selected trackers do not have required custom fields!"
```
Comment (sanak) [inline config/locales/en.yml]: @nobnisai `ensure hat` may be typo. Probably, `ensure that` ?

File: config/locales/ja.yml (hunk 1 above)
Comment (sanak) [inline config/locales/ja.yml]: TODO: @sanak will update Japanese part.

File: config/locales/en.yml (hunk 2 above)
Comment (sanak) [reply config/locales/en.yml]: Thanks for fix!

**Comment by sanak** [comment]
@nobnisai (CC: @dkastl)
Could you update the followings ?
- Delete `.DS_Store` file from this PR branch and add `.gitignore` with having `.DS_Store` line.
- Update `README.md` - `How to use` section with adding new endpoint.
  - Maybe better to change description and existing endpoints titles.

I will update the followings as another PR, then ask you to review it, after merging this PR branch.
- Translate Japanese words.
- Update `README.md`
  - Update `Requirements` section. (Redmine >= ~~4.0.0~~ 4.2.0)
  - Update `Version history` section. (Add this functionality)
  - Add your name to `Authors` section.

**Comment by nobnisai** [comment]
> @nobnisai (CC: @dkastl) Could you update the followings ?
> 
> * Delete `.DS_Store` file from this PR branch and add `.gitignore` with having `.DS_Store` line.
> * Update `README.md` - `How to use` section with adding new endpoint.
>   
>   * Maybe better to change description and existing endpoints titles.
> 
> I will update the followings as another PR, then ask you to review it, after merging this PR branch.
> 
> * Translate Japanese words.
> * Update `README.md`
>   
>   * Update `Requirements` section. (Redmine >= ~4.0.0~ 4.2.0)
>   * Update `Version history` section. (Add this functionality)
>   * Add your name to `Authors` section.

I added the new endpoint in `README.md` - `How to use` section and delete the `.DS_Store` file

**Comment by sanak** [comment]
@nobnisai (CC: @dkastl)
Thanks for the quick fix!

I will merge this PR now.

**Pull request merged**